Jenkinsfile

```diff
@@ -0,0 +1,9 @@
+defaultCiPipeline {
+	platform = ['ubuntu']
+	ciBuildDockerfile = 'javascript.Dockerfile'
+
+	packageId = 'wallet-desktop'
+
+	publisher = 'docker'
+	dockerImageName = 'symbolplatform/symbol-desktop-wallet'
```

Comment:
enable code coverage?

Reply:
if that's okay, keen to leave that one for the next PR because need to find a way to integrate `nyc` for the coverage threshold check.